dev-libs/tree-sitter-meta/tree-sitter-meta-2.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Metapackage for Tree Sitter packages"
+HOMEPAGE="https://github.com/tree-sitter/tree-sitter"
+
+LICENSE="metapackage"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	dev-libs/tree-sitter
+	dev-libs/tree-sitter-bash
+	dev-libs/tree-sitter-c
+	dev-libs/tree-sitter-c-sharp
```

Comment:
Sorry, why do we need -2?

Reply:
should I change it to `-r1`?

Reply:
I think I like r1 more but I don't know what's standard for meta packages

Reply:
fixed